src/xdocs/config_coding.xml

```diff
@@ -3580,6 +3580,109 @@ System.out.println(s2 instanceof Square); //true
       </subsection>
     </section>
 
+    <section name="NoEnumTrailingComma">
+      <p>Since Checkstyle 8.28</p>
+      <subsection name="Description" id="NoEnumTrailingComma_Description">
+        <p>
+          Checks that enum definition does not contain a trailing comma.
+        </p>
```

Comment:
Please add a short rationale as you did it [here](https://github.com/checkstyle/checkstyle/pull/7228/files#diff-5515673b961eef4c335978be88aa4ca1R27)

Reply:
Done